litmus-portal/frontend/src/pages/WorkflowInfoStats/WorkflowRunTable/TableToolbar.tsx

```diff
@@ -0,0 +1,242 @@
+import {
+  Button,
+  FormControl,
+  IconButton,
+  InputAdornment,
+  InputBase,
```

Comment:
can you switch to litmus-ui inputfield

Reply:
Design (Old):
![Screenshot from 2021-07-02 17-49-40](https://user-images.githubusercontent.com/40641427/124273666-ed9f1780-db5d-11eb-929c-7380f32f9fb2.png)

Litmus UI:
![Screenshot from 2021-07-02 17-50-25](https://user-images.githubusercontent.com/40641427/124273830-24752d80-db5e-11eb-841e-5f5376566690.png)

We will be able to remove this one as we migrate to the data grid component...

Reply:
There is a search component in litmus-ui @vanshBhatia-A4k9